src/utils/derivationPath.js

```diff
@@ -0,0 +1,63 @@
+import { ChainId } from '@liquality/cryptoassets'
+import { ChainNetworks } from '@/utils/networks'
+import { BTC_ADDRESS_TYPE_TO_PREFIX } from '@/utils/address'
+import { bitcoin } from '@liquality/types'
+import { LEDGER_BITCOIN_OPTIONS } from '@/utils/ledger-bridge-provider'
+
+const getBitcoinDerivationPath = (accountType, coinType, index) => {
+  if (accountType.includes('bitcoin_ledger')) {
+    const option = LEDGER_BITCOIN_OPTIONS.find(o => o.name === accountType)
+    const { addressType } = option
+    return `${BTC_ADDRESS_TYPE_TO_PREFIX[addressType]}'/${coinType}'/${index}'`
+  } else {
+    return `${BTC_ADDRESS_TYPE_TO_PREFIX[bitcoin.AddressType.BECH32]}'/${coinType}'/${index}'`
+  }
+}
+
+const getEthereumBasedDerivationPath = (coinType, index) => `m/44'/${coinType}'/${index}'/0/0`
```

Comment:
`getEthereumBaseDerivationPath`?

Reply:
should be re-used for all the EVM based chains